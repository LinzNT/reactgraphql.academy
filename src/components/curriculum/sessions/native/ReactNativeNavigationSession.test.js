import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ReactNativeNavigationSession from './ReactNativeNavigationSession'

describe('<ReactNativeNavigationSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ReactNativeNavigationSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
