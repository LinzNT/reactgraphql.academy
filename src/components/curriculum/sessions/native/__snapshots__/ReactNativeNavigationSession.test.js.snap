// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactNativeNavigationSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Types of navigations
    </styled.li>
    <styled.li>
      react-navigation
    </styled.li>
  </styled.ul>
</Session>
`;
